src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,30 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String bulldozerCustom1;
+    private String bulldozerCustom2;
```

Comment:
not informative variable names (fix, please, in other classes too)
```suggestion
    private int horsepower;
    private String model;
    ....
```